### DISCUSSION TITLE
如何让初九生猴子？

### DISCUSSION THREAD

**Discussion by charleszh-dev** [MrChuJiu/Easy.Core.Flow#4] [General 聊天]
😍

**Comment by MrChuJiu** [comment]
首先需要一个母的

**Reply by charleszh-dev** [reply]
![E8C34AEE-FEAC-423C-A301-34B38CBB3674](https://user-images.githubusercontent.com/13272730/138431041-db54a89c-470a-4df5-bfaa-dcc84769fd25.gif)

**Reply by MrChuJiu** [reply]
这个要是给我我能造一车
